src/rpc/RPCHelpers.cpp

```diff
@@ -1327,4 +1329,56 @@ toJsonWithBinaryTx(data::TransactionAndMetadata const& txnPlusMeta, std::uint32_
     return obj;
 }
 
+std::expected<ripple::STArray, Status>
+fetchCredentials(
```

Comment:
Just noticed this function is used in [Deposit_authorized](https://github.com/XRPLF/clio/pull/1712/files#diff-55fbb4e01cb85ae75c92a13d5d06dc8be977bf9f132302fe95f8c1339af6870bR90) RPC call not in LedgerEntry, so i don't think it can go there. It may be used by other RPC methods if they require to retrieve credentials